From 8da4dde42cc25294819ad078432d85437e4a12ee Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Sun, 22 Apr 2018 15:05:46 +0200
Subject: [PATCH] pktsetup/pktsetup.c: do not include <bits/types.h>

This header is not a standard header, and is for example not provided
by the musl C library.

This change has been tested by building udftools against glibc, uClibc
and musl.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Upstream-status: https://github.com/pali/udftools/pull/16
---
 pktsetup/pktsetup.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pktsetup/pktsetup.c b/pktsetup/pktsetup.c
index 8b3df51..81ed142 100644
--- a/pktsetup/pktsetup.c
+++ b/pktsetup/pktsetup.c
@@ -27,7 +27,6 @@
 #include <sys/stat.h>
 #include <unistd.h>
 #include <getopt.h>
-#include <bits/types.h>
 #include <sys/types.h>
 #include <string.h>
 #include <limits.h>
-- 
2.14.3

